# HG changeset patch
# User jwe
# Date 862811815 0
#      Mon May 05 05:56:55 1997 +0000
# Node ID 32186c9020162d6efcb6ecaae8866ed2d8ffbd16
# Parent  295f037b4b3e7cab81381044a019e99c18481e68
[project @ 1997-05-05 05:56:55 by jwe]
Makefile.in

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,16 @@
+Mon May  5 00:54:03 1997  John W. Eaton  <jwe@bevo.che.wisc.edu>
+
+	* DLD-FUNCTIONS, OPERATORS, and	TEMPLATE-INST: New subdirectories.
+	Move appropriate files to new directories.
+	* Makefile.in: Add DLD-FUNCTIONS, OPERATORS, and TEMPLATE-INST
+	directories to VPATH.  Fix rules to work with new directory
+	structure.
+
 Sun May  4 22:40:45 1997  John W. Eaton  <jwe@bevo.che.wisc.edu>
 
 	*  input.cc (initialize_command_input): Rename from
 	initialize_readline.
 	(gnu_readline, octave_gets, get_user_input): Simplify, return
 	string, not char *.
 
 	* Many of other files: Miscellaneous changes to go along with the
